Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -411,35 +413,35 @@ - (void)sizeNavigationContainer {
         }
     }
     
-    backButton.frame = CGRectMake(leadingPadding,
+    self.backButton.frame = CGRectMake(leadingPadding,
```

Comment:
**Nipticky:** Mind reviewing the left indentation on the newly indented lines?

Reply:
Sorry! I'm not exactly clear on what the preferred style is for this one. I've never manually indented my code files like this (just long lines and soft-wrap for me). I'm not seeing an example of this in the style guide either. Is this what we want here?

![Example-Indents](https://user-images.githubusercontent.com/40267301/81889103-58496880-95d5-11ea-9bac-f6e8d0cce253.png)

Reply:
Thank you for fixing this!! Yeah as a rule of thumb (unless it's documented elsewhere) I just follow whatever Xcode's default behavior is, when you hit `return`.

Looks great!